tests/main/install-hook/task.yaml

```diff
@@ -0,0 +1,61 @@
+summary: Check install and remove hooks.
+
+prepare: |
+    echo "Build basic test package"
+    snapbuild $TESTSLIB/snaps/snap-install-hooks .
+    snapbuild $TESTSLIB/snaps/snap-install-hooks-broken1 .
+  
+restore: |
+    rm snap-install-hooks_1.0_all.snap
+    rm snap-install-hooks-broken1_1.0_all.snap
+
+execute: |
+    echo "Installing a snap with hooks"
+    snap install --dangerous snap-install-hooks_1.0_all.snap
+
+    echo "Verify configuration value with snap get"
+    snap get snap-install-hooks installed | MATCH 1
+    snap get snap-install-hooks foo | MATCH bar
```

Comment:
Good catch, I didn't commit 'configure' hook which does that (or rather, git ignored it silently due to my .gitignore ;)). Fixed now.